pkg/minikube/cruntime/cri.go

```diff
@@ -84,6 +384,22 @@ image-endpoint: unix://{{.Socket}}
 	return cr.Run(fmt.Sprintf("sudo mkdir -p %s && printf %%s \"%s\" | sudo tee %s", path.Dir(cPath), b.String(), cPath))
 }
 
+// generateCRIOConfig sets up /etc/crio/crio.conf
+func generateCRIOConfig(cr CommandRunner, k8s config.KubernetesConfig) error {
+	cPath := constants.CRIOConfFile
+	t, err := template.New("crio.conf").Parse(crioConfigTemplate)
+	if err != nil {
+		return err
+	}
+	podInfraContainerImage, _ := images.CachedImages(k8s.ImageRepository, k8s.KubernetesVersion)
```

Comment:
suggestion for a future optional refactor: add `PauseImage` function to the `images` function that returns just the `podInfraContainerImage`. It seems that all the users of `CachedImages` only care either about the single pause image, or all the images, but never both.

Reply:
Done